src/main/webapp/WEB-INF/tags/instructor/home/search.tag

```diff
@@ -13,7 +13,7 @@
 			               class="form-control"
 			               data-toggle="tooltip"
 			               data-placement="top"
-			               placeholder="Student Name">
+			               placeholder="e.g. Charles Shultz, charles@gmail.tmt">
```

Comment:
Ah right, that, I saw gmail and assumed it was .com, @ngzhian could you change that? :)